### ISSUE TITLE
Translating C to V should mention clang is required

### ISSUE THREAD

**Issue by FritteX14** [vlang/v#15783]
**V version:** V 0.3.1 7e69619.0e49ce4
**OS:** linux, Pop!_OS 22.04 LTS

**What did you do?**
I tried to translate this C code:
```
#include <stdio.h>

int main() {
    for (int i = 0; i < 11; i++) {
        printf("Hello World!\n");
    }
    return 0;
}
```
to V using `v translate test.c` as in the [Official Documentation](https://github.com/vlang/v/blob/master/doc/docs.md#translating-c-to-v) but it gave me an error.

**What did you expect to see?**
Translated V code.
 
**What did you see instead?**
```
================ V panic ================
   module: main
 function: find_clang_in_path()
  message: cannot find clang in PATH
     file: ./c2v.v:96
   v hash: 7e69619
=========================================
/tmp/v_1000/../../../../../../home/frittex/Desktop/Programieren/V/v/vlib/builtin/builtin.c.v:53: at panic_debug: Backtrace
/tmp/v_1000/../../../../../../home/frittex/.vmodules/c2v/c2v.v:96: by main__find_clang_in_path
/tmp/v_1000/../../../../../../tmp/v_1000/c2v.tmp.c:17579: by _vinit
/tmp/v_1000/../../../../../../tmp/v_1000/c2v.tmp.c:17591: by main
C2V command: '/home/frittex/.vmodules/c2v/c2v' 'test.c'
C2V failed to translate the C files. Please report it via GitHub.
```

**Comment by JalonSolov** [comment]
Do you have clang installed?  Works fine on Linux, as long as you do.
```
$ v translate hello.c
C to V translator 0.3.1
  translating hello.c         ... Reformatted file: /tmp/hello.v
 took   453 ms ; output .v file: hello.v
Translated   1 files in   453 ms.
$ v run hello.v
Hello World!
Hello World!
Hello World!
Hello World!
Hello World!
Hello World!
Hello World!
Hello World!
Hello World!
Hello World!
Hello World!
$ clang --version
clang version 14.0.6
Target: x86_64-pc-linux-gnu
Thread model: posix
InstalledDir: /usr/bin
$ v version
V 0.3.1 0e49ce4
$
```

**Comment by splace** [comment]
i initially got this due to not realising clang was needed.

**Comment by JalonSolov** [comment]
I see.  Even on the main https://github.com/vlang/c2v site, this is only mentioned in a "Note:" fairly far down the page.  It should be more prominent that clang is required.

**Comment by FritteX14** [comment]
After installing clang with `sudo apt install clang` everything works as intended.

**Comment by Wertzui123** [comment]
I think this can be closed now due to #15802 being merged.
